stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -43,7 +43,14 @@ def _compute_procure_recommended(self):
         # '_quantity_in_progress' override in 'purchase_stock' method has not
         # been designed to work with NewIds (resulting in KeyError exceptions).
         # To circumvent this, we knowingly skip such records here.
-        op_qtys = self.filtered(lambda x: x.id)._quantity_in_progress()
+        ops = self.filtered(lambda x: x.id)
```

Comment:
You can use the short writing:
```suggestion
        ops = self.filtered("id")
```

Reply:
@rousseldenis fixed, can you update your review?